pulsar-client-cpp/lib/Commands.cc

```diff
@@ -36,6 +37,52 @@ using namespace pulsar::proto;
 
 DECLARE_LOG_OBJECT();
 
+static inline bool isBuiltInSchema(SchemaType schemaType) {
+    switch (schemaType) {
+        case STRING:
+        case JSON:
+        case AVRO:
+        case PROTOBUF:
+            return true;
+
+        default:
+            return false;
+    }
+}
+
+static inline proto::Schema_Type getSchemaType(SchemaType type) {
+    switch (type) {
+        case SchemaType::NONE:
+            return Schema_Type_None;
+        case STRING:
+            return Schema_Type_String;
+        case JSON:
+            return Schema_Type_Json;
+        case PROTOBUF:
+            return Schema_Type_Protobuf;
+        case AVRO:
+            return Schema_Type_Avro;
+        default:
```

Comment:
Why `Schema_Type_None` for other schema types?

Reply:
I think I missed several of them here. Will fix

Reply:
Actually, these are the only types defined in the wire proto: 

https://github.com/apache/pulsar/blob/master/pulsar-common/src/main/proto/PulsarApi.proto#L26-L32